app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -619,12 +602,93 @@ class MainActivity : AppCompatActivity(), MainViewController, RouteCallback,
         searchResultsView.onSearchResultsSelectedListener = this
     }
 
+    private fun baseAttributionParams(): RelativeLayout.LayoutParams {
```

Comment:
I know, its pretty unfortunate :/

We could do this but it wouldn't save us much code and actually in the case where we have multiple search results (and we show the text indicator above our pager) we will have to still adjust the attribution text separately. We could/should add a method as part of the sdk to make this less painful for EM and for other developers though.